### DISCUSSION TITLE
Welcome to test-for-learning-git Discussions!

### DISCUSSION THREAD

**Discussion by Pengyu-Jin** [Pengyu-Jin/test-for-learning-git#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pengyu-Jin** [comment]
that's a good start, pengyu

**Comment by GyroJone** [comment]
it's your test to get a achievement right??

**Comment by Pengyu-Jin** [comment]
yes can you help me ?

**Comment by GyroJone** [comment]
of course my friend🥰
